custom/icds_reports/utils.py

```diff
@@ -422,3 +425,29 @@ def chosen_filters_to_labels(config, default_interval=''):
         .format(gender=chosen_gender, delimiter=delimiter, age=chosen_age) if gender or age else ''
 
     return gender_label, age_label, chosen_filters
+
+
+def zip_folder(root_path, name_of_folder):
+    folder_path = os.path.join(root_path, name_of_folder)
+    contents = os.walk(folder_path)
+    output = os.path.join(root_path, '{}.zip'.format(name_of_folder))
+    zip_file = zipfile.ZipFile(output, 'w', zipfile.ZIP_DEFLATED)
+    for root, folders, files in contents:
+        for file_name in files:
+            absolute_path = os.path.join(root, file_name)
+            zip_file.write(absolute_path, file_name)
+    zip_file.close()
+
+
+def create_pdf_file(file_name, directory, pdf_context):
+    template = get_template("icds_reports/icds_app/pdf/issnip_monthly_register.html")
+    file_name = "{}.pdf".format(file_name)
+    resultFile = open(os.path.join(directory, file_name), "w+b")
+    pisa.CreatePDF(
+        template.render(pdf_context),
+        dest=resultFile)
+    resultFile.close()
+
+
+def have_access_to_features(user):
```

Comment:
nit: but I think there could be a better name for this. like `icds_pre_release_features` or something.